test/integration/master/kms_transformation_test.go

```diff
@@ -283,7 +283,7 @@ func mustBeUnHealthy(t *testing.T, checkName string, clientConfig *rest.Config)
 	})
 
 	if pollErr == wait.ErrWaitTimeout {
```

Comment:
I do think we still need to assign `restErr = err` inside the loop above

Reply:
yeah，I agree，now i fixed it.